src/mixins/document-mixin.ts

```diff
@@ -30,11 +30,12 @@ export default class DocumentMixin extends Vue {
         }
         return data
       }).catch(error => {
+        console.log('Get presigned url error =', error) // eslint-disable-line no-console
         throw error
```

Comment:
Catching and then re-throwing an error does nothing, but this was an opportunity to display the error message (since I am not sure the caller handles it well).

Reply:
yeah, the error thrown isn't currently handled by the various upload components.  there's a ticket for this and it's something I've been wanting to fix at some point.